client/src/app/pipes/reverse.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import {Pipe, PipeTransform} from "@angular/core";
+import {Observable} from "rxjs";
+import 'rxjs/add/operator/map';
+
+@Pipe({
+  name: 'reverse'
+})
+
+export class ReversePipe implements PipeTransform{
+  transform($value: Observable<Array<any>>){
+    return $value.map(item => item.slice().reverse());
```

Comment:
reverse mógłbym również użyć bezpośrednio w przypisaniu do listy w konstruktorze create
`this.wordListService.wordList$.subscribe( results =>{
      this.wordList = results.slice().reverse();
    });`

w tedy nie byłoby potrzeby konwertowania w templacie Observable przez async. Powyższy commit daję nam ogólnie sortowanie w templatach, to taka funckjoalność frontedowa bliżej, bo nie chcemy tutaj robić reversu na oryginalnej tablicy